src/config/recommended.ts

```diff
@@ -0,0 +1,50 @@
+import { LintRawConfig } from './config';
+
+export default {
+  rules: {
+    'info-description': 'warn',
+    'info-contact': 'off',
+    'info-license': 'off',
+    'info-license-url': 'off',
+
+    'tag-description': 'warn',
+    'tags-alphabetical': 'off',
+
+    'parameter-description': 'off',
+    'no-path-trailing-slash': 'error',
+    'no-ambiguous-paths': 'warn',
+    'path-declaration-must-exist': 'error',
+    'path-not-include-query': 'error',
+    'path-parameters-defined': 'error',
+    'operation-description': 'off',
+    'operation-2xx-response': 'warn',
+    'operation-operationId': 'off',
```

Comment:
I think this should be error or warning. Having an `operationId` improves so much about the OpenAPI definition's usability.

Reply:
And because this is "recommended" I would actually probably make it an `error`.